rust/src/pgsql/pgsql.rs

```diff
@@ -356,9 +376,22 @@ impl PgsqlState {
                         _needed,
                         needed_estimation
                     );
+                    SCLogDebug!("PgsqlEvent::TruncatedData");
+                    self.set_event(PgsqlEvent::TruncatedData);
```

Comment:
I do not think we should set an event when a PGSQL payload is fragmented over multiple TCP packets...

Reply:
Oki 🙇🏽

Reply:
(As it is legit traffic)